homeassistant/components/whois/sensor.py

```diff
@@ -14,16 +13,16 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
 
-_LOGGER = logging.getLogger(__name__)
-
-DEFAULT_NAME = "Whois"
-
-ATTR_EXPIRES = "expires"
-ATTR_NAME_SERVERS = "name_servers"
-ATTR_REGISTRAR = "registrar"
-ATTR_UPDATED = "updated"
+from .const import (
+    ATTR_EXPIRES,
+    ATTR_NAME_SERVERS,
+    ATTR_REGISTRAR,
+    ATTR_UPDATED,
+    DEFAULT_NAME,
+    LOGGER,
+)
 
-SCAN_INTERVAL = timedelta(hours=24)
+SCANTERVAL = timedelta(hours=24)
```

Comment:
When is this constant used?

Reply:
Nowhere, old-school platform magic :)
The core picks it up.

Reply:
The change is a typo though.

Reply:
Aah darn, yes. Will adjust in the follow ups

Reply:
Added fix to #63069